modules/Students/student_view_details.php

```diff
@@ -657,45 +677,6 @@
                             echo '</ul>';
```

Comment:
Now that Teachers, Tutors, Educational Assistants and Head of Year are in the mix, would it be helpful to indicate who they are in this list? At TIS we're running a modified version of this code that includes the class name next to the teacher. In this case, it could say the class name, or their role (Educational Assistant, Head of Year, etc.)

Reply:
That is an interesting idea. One aim here was that you could copy and paste the whole list straight into an email client. Does your modified code support that? If so, it would be worth adding it in. Thanks!

Reply:
Good question! I guess since modding that part many years ago no one here has been using it that way 😁 I’ll take a look and see if there’s a way to get the best of both worlds. Otherwise, looks great, going to squash and merge 👍